frameworks/helloworld/src/main/dist/examples/overlay.yml

```diff
@@ -38,12 +38,14 @@ pods:
         cpus: {{HELLO_CPUS}}
         memory: {{HELLO_MEM}}
         ports:
-          test:
-            port: 4044
+          dummy:
+            port: 1025
+          dynport:
+            port: 0
     tasks:
       server:
         goal: RUNNING
-        cmd: "python3 http_py3responder.py 4044 hello from the overlay network!"
+        cmd: "python3 http_py3responder.py $PORT_DYNPORT hello from the overlay network!"
```

Comment:
Yes it is. When assigning dynamic ports on a virtual network that does not depend on Mesos resources, it should start at 1025:
https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java#L29